Add implementation of a user queue

LeaPhant opened the pull request.

LeaPhant committed:
Add queue

LeaPhant committed:
Escape indentation hell

respektive commented on src/index.js:
```diff
@@ -13,6 +13,36 @@ const runSql = util.promisify(connection.query).bind(connection)
 const port = config.PORT
 
+
+const MAX_ACTIVE = 2;
```
could probably be a config option. i think 2 is quite low, i would probably set it to like 5 maybe. :thinking:

LeaPhant committed:
Use config option for max active users
resolves https://github.com/respektive/osualt-score-fetcher-server/pull/2/files#r1896508357

respektive reviewed (approved):
lgtm, thanks!
